components/backup/src/utils.rs

```diff
@@ -90,11 +90,11 @@ pub fn create_tokio_runtime(thread_count: usize, thread_name: &str) -> TokioResu
         .thread_name(thread_name)
         .enable_io()
         .enable_time()
-        .on_thread_start(|| {
+        .after_start_wrapper(|| {
```

Comment:
Let's use a warper to wrap around these builder, and override the definition of the `after_start` and `before_start`

Reply:
and then utilize `disallowed_types` to avoid using the original builders

Reply:
I tried this and found a problem with it.
If we call a method not overridden, we could get the original builder inside the wrapper, and then we are unable to call the method which is overridden.